controler/usersControler.js

```diff
@@ -0,0 +1,43 @@
+const peopleModel = require("../models/People");
+const getUsers = (req, res, next) => {
+  res.render("users");
+};
+
+const createUser = async (req, res, next) => {
+  try {
+    const peopleInfo = await peopleModel.find();
+    if(peopleInfo== []){
+            console.log("NO data found!");
```

Comment:
Additional thing: It's totally optional js log stream is not much good. Instead, you can you some logger like (`pino-express-logger` / `winston`)